COPYING-LGPLV3

```diff
@@ -0,0 +1,165 @@
+                   GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
Has there been a decision on the license yet? If not, dont include this for now. But, if there is, make sure to include the heading of the license in each source file (not absolutely needed in configfiles, examples, makefiles, etc).